test/unit/puppetlabs/services/jruby/jruby_puppet_core_test.clj

```diff
@@ -14,6 +14,7 @@
    {:name "puppetserver", :update-server-url "http://localhost:11111"},
    :jruby-puppet
    {:gem-home "./target/jruby-gem-home",
+    :gem-path "./target/jruby-gem-home:./target/vendored-jruby-gems"
```

Comment:
Hmm, I was going to suggest changing the value for `:gem-path` into an array to match elsewhere.  I can't find any code that actually references this function, though.  So maybe `min-config` should just go away completely?